apply-roles

```diff
@@ -159,12 +156,13 @@ main() {
   # retrieve Service principal id
   service_principal_id=$(az ad sp show --id "${application_id}" --query "objectId" --output tsv)
 
-  echo "Assigning roles to subscription(s):"
   if [[ -n "${subscription_id}" ]]; then
+    echo "Assigning roles to subscription:"
     # A subscription id was passed in so only apply to that subscription
     apply_roles_to_subscription "${subscription_id}" "${service_principal_id}"
   else
     # Apply to all subscriptions in Active Directory
+    echo "Assigning roles to ${#subscription_ids_in_active_directory[@]} subscriptions:"
```

Comment:
FYI `${#subscription_ids_in_active_directory[@]}` will display a count of the number of items in the `subscription_ids_in_active_directory` array so this will display:

`Assigning roles to 2 subscriptions:`